doc/source/support/frequently-asked-questions.rst

```diff
@@ -351,18 +348,37 @@ Visit the following URL of your package: ``https://github.com/<org-name>/<packag
 
 Click ``Run workflow`` and select the ``main`` branch. Your online documentation will be updated with the latest changes without a new release.
 
+.. _faq-doc-favicon-logo:
```

Comment:
Instruction for adding favicon and logo. It has not been added to the cookiecutter repo since it should be entirely optional.